cylc/rose/__init__.py

```diff
@@ -73,39 +73,41 @@
 Special Variables
 -----------------
 
-The Cylc Rose plugin has specific logic for handling a small group of special
-environment / template variables in the ``rose-suite.conf`` file:
+The Cylc Rose plugin provides two environment/template variables
+to the Cylc scheduler:
 
 ``ROSE_ORIG_HOST``
-   The plugin provides the hostname of the computer where the plugin runs as
-   an evironment variable.
+   Cylc commands (such as ``cylc validate`` and ``cylc play``)
+   will provide the name of the host on which the script is run,
+   unless the workflow has been installed using ``cylc install``.
+   If this is the case the value set in
+   ``opt/rose-suite-cylc-install.conf`` will be used.
```

Comment:
This makes it sound like `cylc install` does *not* set `ROSE_ORIG_HOST`.

```suggestion
   Cylc commands (such as ``cylc install``, ``cylc validate`` and ``cylc play``)
   will provide the name of the host on which the command is run.

   If the workflow is installed the value of ``ROSE_ORIG_HOST`` will be
   set in ``opt/rose-suite-cylc-install.conf`` and used by future commands
   e.g. ``cylc play``.
```